components/article/stylesheets/index.styl

```diff
@@ -445,6 +445,7 @@ body.body-article
       margin-right 20px
     .article-section-callout
       garamond-size('l-body')
+      width "calc(100% - %s)" % (140px)
```

Comment:
Tried this but stylus doesn't seem to play nice with calc if the units are different